<launch>
  <arg name="mapnum" default="1"/>
  <param name="mapnum" value="$(arg mapnum)"/>
  <arg name="robotnum" default="1"/>
  <param name="robotnum" value="$(arg robotnum)"/>
  <arg name="pathnum" default="1"/>
  <param name="pathnum" value="$(arg pathnum)"/>
  
  <!--arg name="map_file" default="$(env HOME)/owayeol/map$(arg mapnum)/map.yaml"/-->

  <node pkg="owayeol" type="patrol.py" name="patrol" output="screen">
    <remap from="patrol" to="/robot$(arg robotnum)/patrol"/>
    <remap from="/maincommand" 			to="/robot$(arg robotnum)/maincommand" />
    <remap from="/changepath" 			to="/robot$(arg robotnum)/changepath" />
    <remap from="/move_base/cancel" 		to="/robot$(arg robotnum)/move_base/cancel" />
    <remap from="/move_base/result"             to="/robot$(arg robotnum)/move_base/result" />
    <remap from="/move_base_simple/goal"        to="/robot$(arg robotnum)/move_base_simple/goal" />
  </node>
</launch>
 
